docker-compose.demo.yml

```diff
@@ -23,8 +23,8 @@ services:
         environment:
             MAPPED_PORT: 8080
         volumes:
-            - ./ssl/mender-api-gateway/cert.pem:/var/www/mendersoftware/cert/cert.pem
-            - ./ssl/mender-api-gateway/key.pem:/var/www/mendersoftware/cert/key.pem
+            - ./certs/api-gateway/cert.crt:/var/www/mendersoftware/cert/cert.crt
+            - ./certs/api-gateway/private.key:/var/www/mendersoftware/cert/private.key
```

Comment:
@bboozzoo I think this should be ready to merge with https://github.com/mendersoftware/mender-api-gateway-docker/pull/52/files already; they should be mounted to the right place, no?